flutter/test/mocks.mocks.dart

```diff
@@ -0,0 +1,101 @@
+// Mocks generated by Mockito 5.0.0-nullsafety.7 from annotations
```

Comment:
wondering if this should make it to the repo or each local fork should just generate it

Reply:
At least on CI it should be generated so that it is always up to date.

Reply:
Seems that some people suggest check into version control, while others say it can be gitignored

Reply:
lets keep for now and remove later if this becomes the new `lock` file that change in every commit hehe